examples/shfqa_power_spectral_density.md

```diff
@@ -0,0 +1,327 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.14.1
+  kernelspec:
+    display_name: Python 3.9.15 64-bit ('3.9.15')
```

Comment:
```suggestion
    display_name: Python 3 (ipykernel)
```
Just so it is aligned with the other examples. It does not change anything else 😁

Reply:
done.